src/rust/ide/src/double_representation/definition.rs

```diff
@@ -186,6 +165,22 @@ impl DefinitionName {
         };
         Some(DefinitionName {extended_target,name})
     }
+
+    /// Checks if the given definition name is a member of given type.
+    pub fn extends_type(&self, parent_name:&str, expected_typename:&str) -> bool {
+        if self.extended_target.is_empty() {
+            parent_name == expected_typename
+        } else {
+            self.explicitly_extends_type(expected_typename)
+        }
+    }
+
+    /// Check if this name is an explicit extension method for given type, like `Int.add`.
```

Comment:
These docs have soem kind of example, but I still don't understand them. (See previous comment)